model/user.model.js

```diff
@@ -9,7 +9,9 @@ class User {
     return new User(options)
   }
 
-  static createFromDB(options = {}) {
+  static createFromDB(options = []) {
```

Comment:
Think of `createManyFromDB` as a separate function which will return array.
We don't want to fetch 0th index of an array every time when it's clear that there is only one record is requested.